sechub-wrapper-prepare/src/test/java/com/mercedesbenz/sechub/wrapper/prepare/modules/PrepareWrapperModuleGitTest.java

```diff
@@ -23,24 +23,38 @@
 import com.mercedesbenz.sechub.test.TestFileWriter;
 import com.mercedesbenz.sechub.wrapper.prepare.cli.PrepareWrapperEnvironment;
 import com.mercedesbenz.sechub.wrapper.prepare.prepare.PrepareWrapperContext;
+import com.mercedesbenz.sechub.wrapper.prepare.upload.FileNameSupport;
+import com.mercedesbenz.sechub.wrapper.prepare.upload.PrepareWrapperUploadService;
 
 class PrepareWrapperModuleGitTest {
 
-    private PrepareWrapperModuleGit moduleToTest;
+    PrepareWrapperModuleGit moduleToTest;
 
-    private WrapperGit git;
+    WrapperGit git;
 
     TestFileWriter writer;
 
     GitInputValidator gitInputValidator;
 
+    PrepareWrapperUploadService uploadService;
+
+    FileNameSupport filesSupport;
+
+    String subfolder = "subfolder";
+
     @BeforeEach
-    void beforeEach() {
+    void beforeEach() throws IOException {
```

Comment:
For tests it is absolutely okay to always throw any kind of exception. I would do always this:
```suggestion
    void beforeEach() throws Exception {
```